contrib/CMakeLists.txt

```diff
@@ -32,6 +32,8 @@ endif()
 
 set_property(DIRECTORY PROPERTY EXCLUDE_FROM_ALL 1)
 
+include_directories("${CMAKE_SOURCE_DIR}/contrib/magic_enum/include")
```

Comment:
I'm not sure if it works here, maybe it's only for descendant CMakeFiles?

Reply:
I guess it doesn't, but I failed the fight with CMake to be able to just `#include<magic_enum.hpp>` in EnumReflection.h, it won't work.

Reply:
Should be doable... most likely we have some other header only libs in contrib.

Reply:
Maybe put into `target_include_directories` for `base`?

Reply:
You can add an INTERFACE library, and link it to the dbms (or clickhouse_common_io), you can take a look at how pcg_random is added - https://github.com/ClickHouse/ClickHouse/blob/3e65fa1a10bc9d489aec536c5b270856b767ac25/base/pcg-random/CMakeLists.txt#L1-L2

Reply:
It worked, thanks!